src/iiif/tools.py

```diff
@@ -364,3 +382,88 @@ def send_email(email_address, email_subject, email_body):
         html_content=email_body
     )
     sg.send(email)
+
+
+def check_zip_payload(payload):
+    if 'urls' not in payload or len(payload['urls']) == 0:
+        raise ImmediateHttpResponse(response=HttpResponse("No urls detected in json", status=400))
+
+
+def strip_full_iiif_url(url):
+    if '/iiif/' not in url:
+        raise ImmediateHttpResponse(response=HttpResponse("Misformed paths", status=400))
+
+    # Strip the domain from the url and return the only relevant part
+    return url.split('/iiif/')[-1]
+
+
+def store_zip_job(zip_info):
+    # There's a lot of things in the request_meta that is not JSON serializable.
+    # Since we just need some headers we simply remove all values that are not strings
+    zip_info['request_meta'] = {k: v for k, v in zip_info['request_meta'].items() if type(v) is str}
```

Comment:
you might want to extend type checking to more serializable data types:

`type(v) in [str, int, bool, float]`

Reply:
We only need a couple strings. So this is fine.